internal/controller/vrg_volrep.go

```diff
@@ -1805,6 +1757,26 @@ func setPVCClusterDataProtectedCondition(protectedPVC *ramendrv1alpha1.Protected
 	}
 }
 
+func (v *VRGInstance) updatePVCLastSyncCounters(pvcNamespace, pvcName string, status *volrep.VolumeReplicationStatus) {
+	protectedPVC := v.findProtectedPVC(pvcNamespace, pvcName)
+	if protectedPVC == nil {
+		return
+	}
+
+	if status == nil {
+		protectedPVC.LastSyncTime = nil
+		protectedPVC.LastSyncDuration = nil
```

Comment:
We do not set or unset `protectedPVC.LastSyncBytes` if PVC is volsync protected, as vlsync does not report this data. I would prefer if there was a latch here as well like in line 1774 below (even though the default is nil, for future awareness)

Reply:
Done